### ISSUE TITLE
2025-01-17, Papers for large language model, llm

### ISSUE THREAD

**Issue by 9tailwolf** [Arxiv-Sender/Arxiv-Sender#54]
# Today's Arxiv Papers

(1) [Distilling Multi-modal Large Language Models for Autonomous Driving](http://arxiv.org/abs/2501.09757v1), 2025-01-16

(2) [Lost in Translation, Found in Context: Sign Language Translation with Contextual Cues](http://arxiv.org/abs/2501.09754v1), 2025-01-16

(3) [OmniThink: Expanding Knowledge Boundaries in Machine Writing through Thinking](http://arxiv.org/abs/2501.09751v1), 2025-01-16

(4) [Enhancing Lexicon-Based Text Embeddings with Large Language Models](http://arxiv.org/abs/2501.09749v1), 2025-01-16

(5) [Suggesting Code Edits in Interactive Machine Learning Notebooks Using Large Language Models](http://arxiv.org/abs/2501.09745v1), 2025-01-16

(6) [Inference-Time Scaling for Diffusion Models beyond Scaling Denoising Steps](http://arxiv.org/abs/2501.09732v1), 2025-01-16

(7) [CyberMentor: AI Powered Learning Tool Platform to Address Diverse Student Needs in Cybersecurity Education](http://arxiv.org/abs/2501.09709v1), 2025-01-16

(8) [Domain Adaptation of Foundation LLMs for e-Commerce](http://arxiv.org/abs/2501.09706v1), 2025-01-16

(9) [Simulated Interactive Debugging](http://arxiv.org/abs/2501.09694v1), 2025-01-16

(10) [Towards Large Reasoning Models: A Survey of Reinforced Reasoning with Large Language Models](http://arxiv.org/abs/2501.09686v1), 2025-01-16

(11) [Robin: a Suite of Multi-Scale Vision-Language Models and the CHIRP Evaluation Benchmark](http://arxiv.org/abs/2501.09672v1), 2025-01-16

(12) [A Survey of Research in Large Language Models for Electronic Design Automation](http://arxiv.org/abs/2501.09655v1), 2025-01-16

(13) [The Heap: A Contamination-Free Multilingual Code Dataset for Evaluating Large Language Models](http://arxiv.org/abs/2501.09653v1), 2025-01-16

(14) [CarMem: Enhancing Long-Term Memory in LLM Voice Assistants through Category-Bounding](http://arxiv.org/abs/2501.09645v1), 2025-01-16

(15) [LLM-Based Routing in Mixture of Experts: A Novel Framework for Trading](http://arxiv.org/abs/2501.09636v1), 2025-01-16

(16) [Empowering Large Language Models in Wireless Communication: A Novel Dataset and Fine-Tuning Framework](http://arxiv.org/abs/2501.09631v1), 2025-01-16

(17) [Beyond Reward Hacking: Causal Rewards for Large Language Model Alignment](http://arxiv.org/abs/2501.09620v1), 2025-01-16

(18) [From Scarcity to Capability: Empowering Fake News Detection in Low-Resource Languages with LLMs](http://arxiv.org/abs/2501.09604v1), 2025-01-16

(19) [AI in Support of Diversity and Inclusion](http://arxiv.org/abs/2501.09534v1), 2025-01-16

(20) [AdaFV: Accelerating VLMs with Self-Adaptive Cross-Modality Attention Mixture](http://arxiv.org/abs/2501.09532v1), 2025-01-16

(21) [Confidence Estimation for Error Detection in Text-to-SQL Systems](http://arxiv.org/abs/2501.09527v1), 2025-01-16

(22) [Augmenting a Large Language Model with a Combination of Text and Visual Data for Conversational Visualization of Global Geospatial Data](http://arxiv.org/abs/2501.09521v1), 2025-01-16

(23) [Omni-Emotion: Extending Video MLLM with Detailed Face and Audio Modeling for Multimodal Emotion Analysis](http://arxiv.org/abs/2501.09502v1), 2025-01-16

(24) [Evaluating Conversational Recommender Systems with Large Language Models: A User-Centric Evaluation Framework](http://arxiv.org/abs/2501.09493v1), 2025-01-16

(25) [Exploring the Inquiry-Diagnosis Relationship with Advanced Patient Simulators](http://arxiv.org/abs/2501.09484v1), 2025-01-16

(26) [Guided Debugging of Auto-Translated Code Using Differential Testing](http://arxiv.org/abs/2501.09475v1), 2025-01-16

(27) ["A Great Start, But...": Evaluating LLM-Generated Mind Maps for Information Mapping in Video-Based Design](http://arxiv.org/abs/2501.09457v1), 2025-01-16

(28) [Solving the unsolvable: Translating case law in Hong Kong](http://arxiv.org/abs/2501.09444v1), 2025-01-16

(29) [A Survey on Responsible LLMs: Inherent Risk, Malicious Use, and Mitigation Strategy](http://arxiv.org/abs/2501.09431v1), 2025-01-16

(30) [AutoCBT: An Autonomous Multi-agent Framework for Cognitive Behavioral Therapy in Psychological Counseling](http://arxiv.org/abs/2501.09426v1), 2025-01-16

(31) [FASP: Fast and Accurate Structured Pruning of Large Language Models](http://arxiv.org/abs/2501.09412v1), 2025-01-16

(32) [MoE$^2$: Optimizing Collaborative Inference for Edge Large Language Models](http://arxiv.org/abs/2501.09410v1), 2025-01-16

(33) [Evaluating LLM Abilities to Understand Tabular Electronic Health Records: A Comprehensive Study of Patient Data Extraction and Retrieval](http://arxiv.org/abs/2501.09384v1), 2025-01-16

(34) [Adaptive Contextual Caching for Mobile Edge Large Language Model Service](http://arxiv.org/abs/2501.09383v1), 2025-01-16

(35) [Aligning Instruction Tuning with Pre-training](http://arxiv.org/abs/2501.09368v1), 2025-01-16

(36) [PICE: A Semantic-Driven Progressive Inference System for LLM Serving in Cloud-Edge Networks](http://arxiv.org/abs/2501.09367v1), 2025-01-16

(37) [YETI (YET to Intervene) Proactive Interventions by Multimodal AI Agents in Augmented Reality Tasks](http://arxiv.org/abs/2501.09355v1), 2025-01-16

(38) [Rational Tuning of LLM Cascades via Probabilistic Modeling](http://arxiv.org/abs/2501.09345v1), 2025-01-16

(39) [SOP-Agent: Empower General Purpose AI Agent with Domain-Specific SOPs](http://arxiv.org/abs/2501.09316v1), 2025-01-16

(40) [A Study of In-Context-Learning-Based Text-to-SQL Errors](http://arxiv.org/abs/2501.09310v1), 2025-01-16

(41) [RoboReflect: Robotic Reflective Reasoning for Grasping Ambiguous-Condition Objects](http://arxiv.org/abs/2501.09307v1), 2025-01-16

(42) [To Retrieve or Not to Retrieve? Uncertainty Detection for Dynamic Retrieval Augmented Generation](http://arxiv.org/abs/2501.09292v1), 2025-01-16

(43) [LAVCap: LLM-based Audio-Visual Captioning using Optimal Transport](http://arxiv.org/abs/2501.09291v1), 2025-01-16

(44) [Large Language Model is Secretly a Protein Sequence Optimizer](http://arxiv.org/abs/2501.09274v1), 2025-01-16

(45) [Perspective Transition of Large Language Models for Solving Subjective Tasks](http://arxiv.org/abs/2501.09265v1), 2025-01-16

(46) [Delayed Fusion: Integrating Large Language Models into First-Pass Decoding in End-to-end Speech Recognition](http://arxiv.org/abs/2501.09258v1), 2025-01-16

(47) [Clone-Robust AI Alignment](http://arxiv.org/abs/2501.09254v1), 2025-01-16

(48) [Split Fine-Tuning for Large Language Models in Wireless Networks](http://arxiv.org/abs/2501.09237v1), 2025-01-16

(49) [Foundations of Large Language Models](http://arxiv.org/abs/2501.09223v1), 2025-01-16

(50) [A Simple Graph Contrastive Learning Framework for Short Text Classification](http://arxiv.org/abs/2501.09219v1), 2025-01-16

(51) [Interpretable Droplet Digital PCR Assay for Trustworthy Molecular Diagnostics](http://arxiv.org/abs/2501.09218v1), 2025-01-16

(52) [Boosting Short Text Classification with Multi-Source Information Exploration and Dual-Level Contrastive Learning](http://arxiv.org/abs/2501.09214v1), 2025-01-16

(53) [FineMedLM-o1: Enhancing the Medical Reasoning Ability of LLM from Supervised Fine-Tuning to Test-Time Training](http://arxiv.org/abs/2501.09213v1), 2025-01-16

(54) [Personalized Parsons Puzzles as Scaffolding Enhance Practice Engagement Over Just Showing LLM-Powered Solutions](http://arxiv.org/abs/2501.09210v1), 2025-01-16

(55) [Towards Semantics Lifting for Scientific Computing: A Case Study on FFT](http://arxiv.org/abs/2501.09201v1), 2025-01-15

(56) [Guiding Retrieval using LLM-based Listwise Rankers](http://arxiv.org/abs/2501.09186v1), 2025-01-15

(57) [Generative AI Takes a Statistics Exam: A Comparison of Performance between ChatGPT3.5, ChatGPT4, and ChatGPT4o-mini](http://arxiv.org/abs/2501.09171v1), 2025-01-15

(58) [Breaking Barriers or Building Dependency? Exploring Team-LLM Collaboration in AI-infused Classroom Debate](http://arxiv.org/abs/2501.09165v1), 2025-01-15

(59) [The Veln(ia)s is in the Details: Evaluating LLM Judgment on Latvian and Lithuanian Short Answer Matching](http://arxiv.org/abs/2501.09164v1), 2025-01-15

(60) [Evaluating GenAI for Simplifying Texts for Education: Improving Accuracy and Consistency for Enhanced Readability](http://arxiv.org/abs/2501.09158v1), 2025-01-15

(61) [Towards Multilingual LLM Evaluation for Baltic and Nordic languages: A study on Lithuanian History](http://arxiv.org/abs/2501.09154v1), 2025-01-15

(62) [Agentic Retrieval-Augmented Generation: A Survey on Agentic RAG](http://arxiv.org/abs/2501.09136v1), 2025-01-15

(63) [HAFix: History-Augmented Large Language Models for Bug Fixing](http://arxiv.org/abs/2501.09135v1), 2025-01-15

(64) [Multilingual LLMs Struggle to Link Orthography and Semantics in Bilingual Word Processing](http://arxiv.org/abs/2501.09127v1), 2025-01-15

(65) [Augmenting Human-Annotated Training Data with Large Language Model Generation and Distillation in Open-Response Assessment](http://arxiv.org/abs/2501.09126v1), 2025-01-15

(66) [Rethinking Post-Training Quantization: Introducing a Statistical Pre-Calibration Approach](http://arxiv.org/abs/2501.09107v1), 2025-01-15

(67) [Tracking the Takes and Trajectories of English-Language News Narratives across Trustworthy and Worrisome Websites](http://arxiv.org/abs/2501.09102v1), 2025-01-15

(68) [Drama Llama: An LLM-Powered Storylets Framework for Authorable Responsiveness in Interactive Narrative](http://arxiv.org/abs/2501.09099v1), 2025-01-15

(69) [SteLLA: A Structured Grading System Using LLMs with RAG](http://arxiv.org/abs/2501.09092v1), 2025-01-15

(70) [Multimodal LLMs Can Reason about Aesthetics in Zero-Shot](http://arxiv.org/abs/2501.09012v1), 2025-01-15

(71) [Decompose-ToM: Enhancing Theory of Mind Reasoning in Large Language Models through Simulation and Task Decomposition](http://arxiv.org/abs/2501.09056v1), 2025-01-15

(72) [Aegis2.0: A Diverse AI Safety Dataset and Risks Taxonomy for Alignment of LLM Guardrails](http://arxiv.org/abs/2501.09004v1), 2025-01-15

(73) [Development and Validation of the Provider Documentation Summarization Quality Instrument for Large Language Models](http://arxiv.org/abs/2501.08977v1), 2025-01-15

(74) [Learning to Extract Cross-Domain Aspects and Understanding Sentiments Using Large Language Models](http://arxiv.org/abs/2501.08974v1), 2025-01-15

(75) [Analyzing the Ethical Logic of Six Large Language Models](http://arxiv.org/abs/2501.08951v1), 2025-01-15

(76) [Applying General Turn-taking Models to Conversational Human-Robot Interaction](http://arxiv.org/abs/2501.08946v1), 2025-01-15

(77) [Physical AI Agents: Integrating Cognitive Intelligence with Real-World Action](http://arxiv.org/abs/2501.08944v1), 2025-01-15

(78) [Disentangling Exploration of Large Language Models by Optimal Exploitation](http://arxiv.org/abs/2501.08925v1), 2025-01-15

(79) [GenAI Content Detection Task 3: Cross-Domain Machine-Generated Text Detection Challenge](http://arxiv.org/abs/2501.08913v1), 2025-01-15

(80) [Leveraging Large Language Models as Knowledge-Driven Agents for Reliable Retrosynthesis Planning](http://arxiv.org/abs/2501.08897v1), 2025-01-15

(81) [ToMATO: Verbalizing the Mental States of Role-Playing LLMs for Benchmarking Theory of Mind](http://arxiv.org/abs/2501.08838v1), 2025-01-15

(82) [How Developers Interact with AI: A Taxonomy of Human-AI Collaboration in Software Engineering](http://arxiv.org/abs/2501.08774v1), 2025-01-15

(83) [Enhanced Large Language Models for Effective Screening of Depression and Anxiety](http://arxiv.org/abs/2501.08769v1), 2025-01-15

(84) [Leveraging LLM Agents for Translating Network Configurations](http://arxiv.org/abs/2501.08760v1), 2025-01-15

(85) [The Inherent Limits of Pretrained LLMs: The Unexpected Convergence of Instruction Tuning and In-Context Learning Capabilities](http://arxiv.org/abs/2501.08716v1), 2025-01-15

(86) [Knowledge Graph-based Retrieval-Augmented Generation for Schema Matching](http://arxiv.org/abs/2501.08686v1), 2025-01-15

(87) [Augmenting Smart Contract Decompiler Output through Fine-grained Dependency Analysis and LLM-facilitated Semantic Recovery](http://arxiv.org/abs/2501.08670v1), 2025-01-15

(88) [MAGNET: Augmenting Generative Decoders with Representation Learning and Infilling Capabilities](http://arxiv.org/abs/2501.08648v1), 2025-01-15

(89) [Reassessing the Role of Chain-of-Thought in Sentiment Analysis: Insights and Limitations](http://arxiv.org/abs/2501.08641v1), 2025-01-15

(90) [SWSC: Shared Weight for Similar Channel in LLM](http://arxiv.org/abs/2501.08631v1), 2025-01-15

(91) [Disjoint Processing Mechanisms of Hierarchical and Linear Grammars in Large Language Models](http://arxiv.org/abs/2501.08618v1), 2025-01-15

(92) [Assessing the Alignment of FOL Closeness Metrics with Human Judgement](http://arxiv.org/abs/2501.08613v1), 2025-01-15

(93) [Monte Carlo Tree Search for Comprehensive Exploration in LLM-Based Automatic Heuristic Design](http://arxiv.org/abs/2501.08603v2), 2025-01-15

(94) [AutoRestTest: A Tool for Automated REST API Testing Using LLMs and MARL](http://arxiv.org/abs/2501.08600v1), 2025-01-15

(95) [LlamaRestTest: Effective REST API Testing with Small Language Models](http://arxiv.org/abs/2501.08598v1), 2025-01-15

(96) [LoRS: Efficient Low-Rank Adaptation for Sparse Large Language Model](http://arxiv.org/abs/2501.08582v1), 2025-01-15

(97) [What Limits LLM-based Human Simulation: LLMs or Our Design?](http://arxiv.org/abs/2501.08579v1), 2025-01-15

(98) [Information Entropy Invariance: Enhancing Length Extrapolation in Attention Mechanisms](http://arxiv.org/abs/2501.08570v1), 2025-01-15

(99) [Adaptive Sampled Softmax with Inverted Multi-Index: Methods, Theory and Applications](http://arxiv.org/abs/2501.08563v1), 2025-01-15

(100) [Generative Visual Commonsense Answering and Explaining with Generative Scene Graph Constructing](http://arxiv.org/abs/2501.09041v1), 2025-01-15
